shared/handler.go

```diff
@@ -6,14 +6,12 @@ import (
 	"github.com/pokt-network/pocket/shared/types"
 )
 
-// TODO: Only supporting a subset of topics because not all are used.
-func (node *Node) handleEvent(event *types.Event) error {
-	switch event.PocketTopic {
-	case types.ConsensusMessage:
-		node.GetBus().GetConsensusModule().HandleMessage(event.MessageData)
+func (node *Node) handleEvent(event *types.PocketEvent) error {
+	switch event.Topic {
+	case types.PocketTopic_CONSENSUS_MESSAGE_TOPIC:
```

Comment:
As a comment, I think I prefer shorter names. `types.EVENT_TOPIC_CONSENSUS_MESSAGE`. There is quite the redundancy in `PocketTopic_...._TOPIC`.

Reply:
This is a proto generated type from an enum so outside of our control.

Reply:
Yeah correct...